lib/ResponseDefinitions.php

```diff
@@ -37,7 +37,7 @@
  * @psalm-type TablesTable = array{
  * 	id: int,
  * 	title: string,
- * 	emoji: string,
+ * 	emoji: string|null,
  * 	ownership: string,
  * 	ownerDisplayName: string,
```

Comment:
If the type for `ownerDisplayName` is `string|null` in other places, that should be the case here too?